packages/burnside-cli/src/cli.js

```diff
@@ -7,16 +7,14 @@ const argv = require('yargs')
   .alias('s', 'startup')
   .alias('c', 'condition')
   .alias('k', 'karmaConfig')
+  .alias('v', 'verbose')
   .argv;
 
 const main = require('./index.js');
 const prefix = '[Burnside]: ';
-const v = argv.verbose || false;
-const log = str => {
-  if (v) {
-    console.info(prefix + str); // eslint-disable-line
-  }
-};
+
+process.env.trace = Boolean(argv.verbose);
+const log = argv.verbose ? str => console.info(prefix + str) : () => {}; // eslint-disable-line
```

Comment:
Would be nice to see a verbose description of what lint rule is disabled here `/* eslint-disable {SPECIFIC_LINT_RULE}*/`

Reply:
ah yeah I haven't been adhering to that wrt to console logs